ci-operator/step-registry/openshift-qe/rhoso/installer/pre-provisioned/openshift-qe-rhoso-installer-pre-provisioned-ref.yaml

```diff
@@ -0,0 +1,48 @@
+ref:
+  as: openshift-qe-rhoso-installer-pre-provisioned
+  from_image:
+    namespace: ci
+    name: telco-bastion
+    tag: latest
+  commands: openshift-qe-rhoso-installer-pre-provisioned-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  env:
```

Comment:
Most of this should be moved to Vault once we have clarity on which LTA allocation to use

Reply:
if we are fetching it from the vault, no need to mention here?